serve.sh

```diff
@@ -34,7 +34,10 @@ if [[ $* == *--incremental* ]]; then
 fi
 
 # Run the image.
+# Add '-e PAGES_API_URL=https://[domain]/api/v3/ \' if connecting to a
+# GitHub Enterprise AIP repository
 docker run --rm \
+  -e PAGES_REPO_NWO=googleapis/aip \
```

Comment:
This is a work in progress. I found that in order to access Salesforce's GitHub Enterprise account, I had to add the above mentioned environment variable. I have been experimenting with other environment variables as well. It made sense for them to all be in one place, so I moved this here.

Additionally, I think this may free up the option of releasing an `AIP` docker image that can be configured externally rather than needing to be built by each AIP adopter.

A small change for now, but I think may open up larger options moving forward.